request/request.go

```diff
@@ -30,59 +30,59 @@ type CoordinatedRequest struct {
 	State        map[string][]byte `json:"state"`
 	SequenceJSON []byte            `json:"sequence_json,omitempty"`
 
-	bodyValues map[string]interface{} `json:"-"`
+	bodyValues map[string]interface{}
 }
 
-// FromVKRequest creates a CoordinatedRequest from an VK request handler
-func FromVKRequest(r *http.Request, ctx *vk.Ctx) (*CoordinatedRequest, error) {
-	reqBody, err := io.ReadAll(r.Body)
-	if err != nil {
-		return nil, vk.E(http.StatusInternalServerError, "failed to read request body")
+// FromEchoContext creates a CoordinatedRequest from an echo context.
+func FromEchoContext(c echo.Context) (*CoordinatedRequest, error) {
```

Comment:
Make sure to link the corresponding e2core PR when you are done just so its easy to spot when looking at the commit history.